targets/esp8266/jswrap_esp8266.c

```diff
@@ -428,5 +428,35 @@ void jswrap_ESP8266_neopixelWrite(Pin pin, JsVar *jsArrayOfData) {
 
 #endif
 }
+//===== ESP8266.deepSleep
+/*JSON{
+  "type"     : "staticmethod",
+  "class"    : "ESP8266",
+  "name"     : "deepSleep",
+  "generate" : "jswrap_ESP8266_deepSleep",
+  "params"   : [
+    ["micros", "JsVar", "Number of microseconds to sleep."]
+  ]
+
+}*/
+void   jswrap_ESP8266_deepSleep(JsVar *jsMicros) {
+    if (!jsvIsInt(jsMicros)) {
+    jsExceptionHere(JSET_ERROR, "Invalid microseconds.");
+         return;
+    }
+    int sleepTime = jsvGetInteger(jsMicros);
+    system_deep_sleep(sleepTime);
+
+}
```

Comment:
Mhh, this looks identical to https://github.com/tve/Espruino/pull/3/files, would have been nice to mention that it's a copy&paste of someone else's work?